iib/web/api_v1.py

```diff
@@ -147,7 +196,7 @@ def get_artifact_file_from_s3_bucket(
     raise NotFound()
 
 
-def _get_unique_bundles(bundles):
+def _get_unique_bundles(bundles: Optional[List[str]]) -> Optional[List[str]]:
```

Comment:
The parameter isn't optional. It can be an empty list but it will always be passed. Should this be something else?